app/views/problems/_admin.html.erb

```diff
@@ -22,6 +22,15 @@
 <% end %>
 <% if (policy(@problem).use?) && !@problem_sets.nil? && @problem_sets.size>0 %>
   <p>
+  <b> Problem Sets: </b>
+  <ul>
```

Comment:
Should this `<ul>` block only be displayed if `@problem_sets.size > 0`? Also we should be consistent with spacing (around the `==` operator, between here and [here](https://github.com/NZOI/nztrain/pull/26/files#diff-79bf749667c5cfb589135a66fa086e58R46)).

Reply:
Those are good points. I'll fix it up and make a new commit.